provisioning/device/samples/register_tpm.js

```diff
@@ -29,6 +29,7 @@ var idScope = '<the id scope for your dps instance>';
 
 var securityClient = new tpmSecurity.TpmSecurityClient();
 
+var provisioningClient = ProvisioningDeviceClient.create(provisioningHost, idScope, new provisioningTransport(), securityClient);
```

Comment:
nit (fix later): constructors should be pascal-cased (ProvisioningTransport)

Reply:
Indeed.